docs/product-assembly_openshift-ansible-playbook/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,33 @@
+[id='assembly_openshift-immutable]
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Deploying a {PRODUCT} {PRODUCT_VERSION} environment on {OPENSHIFT} using the Ansible playbook
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: openshift-ansible-playbook
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a system engineer, you can deploy a {PRODUCT} environment on {OPENSHIFT} to provide an 
+infrastructure to develop or execute processes and other business assets. You can use
+the Ansible playbook to deploy the environment in an interactive process, setting all
+parameters as necessary.
+
+.Prerequisites
```

Comment:
I think we should mention the pre-provisioning of the PV or the possibility to dynamically provision PVs